edpop_explorer/readers/stcn.py

```diff
@@ -75,6 +75,33 @@ def _wrap_holding(holding_data: dict) -> Field:
     return Field(summary)
 
 
+def _wrap_digitization_from_holding(holding_data: dict) -> Optional[DigitizationField]:
+    # One of the electronicResource and electronicReproduction field may
+    # contain the digitization. See which (if any) is available.
+    # Since the digitizations are connected to holdings and often
+    # come together with a physical holding, we accept that some of the
+    # information is repeated in the holding field.
+    electronic_resource = safeget(holding_data, ("data", "electronicResource"), first=True)
+    electronic_reproduction = safeget(holding_data, ("data", "electronicReproduction"), first=True)
+    if electronic_resource:
+        dig_data = electronic_resource
+    elif electronic_reproduction:
+        dig_data = electronic_reproduction
+    else:
+        return None
```

Comment:
Totally optional, but a nice trick:

```suggestion
    dig_data = electronic_resource or electronic_reproduction
    if not dig_data:
        return None
```